exec-unittests

```diff
@@ -94,25 +126,45 @@ _subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-sc
 let count=0
 for _major in ${major_list}; do
     if [[ "${_major}" == "agent" ]]; then
+        # The parallel program is really cool.  The usage of `parallel` is
+        # internal and automated; we are not disturbing product code to depend
+        # on this tool, and we, as developers, have viewed the citation and are
+        # justified in suppressing future displays of it in our development
+        # processes.
+        #
+        # If `HOME` is not set, the `parallel` program looks here. Don't ask.
+        _para_dir=${HOME:-/tmp}/.parallel
+        mkdir -p ${_para_dir}
+        touch ${_para_dir}/will-cite
```

Comment:
So did `--will-cite` not work? Is there documentation somewhere about creating this `will-cite` marker file? (I didn't find any, although `parallel` seems to try very hard to be opaque about suppressing the nagware.) Otherwise, it seems something like `alias parallel='parallel --will-cite'` would have been cleaner.